docs/spark_expressions_support.md

```diff
@@ -98,7 +98,7 @@
  - [x] arrays_overlap
  - [ ] arrays_zip
  - [x] element_at
- - [ ] flatten
+ - [x] flatten
```

Comment:
this file is autogenerated, probably we need add this to the header

Reply:
Thanks, revert this.